README.md

```diff
@@ -1,79 +1,285 @@
+
+## Issues and projects 
+
+The **Issues** and **Projects** tabs are disabled for this repository as this is part of the Ballerina library. To report bugs, request new features, start new discussions, view project boards, etc., visit the Ballerina library [parent repository](https://github.com/ballerina-platform/ballerina-library). 
+
+This repository only contains the source code for the package.
+
+## Build from the source
+
+### Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 17. You can download it from either of the following sources:
+
+   * [Oracle JDK](https://www.oracle.com/java/technologies/downloads/)
+   * [OpenJDK](https://adoptium.net/)
+
+    > **Note:** After installation, remember to set the `JAVA_HOME` environment variable to the directory where JDK was installed.
+
+2. Download and install [Ballerina Swan Lake](https://ballerina.io/).
+
+3. Download and install [Docker](https://www.docker.com/get-started).
+
+    > **Note**: Ensure that the Docker daemon is running before executing any tests.
+
+### Build options
+
+Execute the commands below to build from the source.
+
+1. To build the package:
+   ```
+   ./gradlew clean build
+   ```
+
+2. To run the tests:
+   ```
+   ./gradlew clean test
+   ```
+
+3. To build the without the tests:
+   ```
+   ./gradlew clean build -x test
+   ```
+
+5. To debug the package with a remote debugger:
+   ```
+   ./gradlew clean build -Pdebug=<port>
+   ```
+
+6. To debug with the Ballerina language:
+   ```
+   ./gradlew clean build -PbalJavaDebug=<port>
+   ```
+
+7. Publish the generated artifacts to the local Ballerina Central repository:
+    ```
+    ./gradlew clean build -PpublishToLocalCentral=true
+    ```
+
+8. Publish the generated artifacts to the Ballerina Central repository:
+   ```
+   ./gradlew clean build -PpublishToCentral=true
+   ```
+
+## Contribute to Ballerina
 
-As an open source project, Ballerina welcomes contributions from the community.
+As an open-source project, Ballerina welcomes contributions from the community.
 
-For more information, go to
-the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
 
 ## Code of conduct
 
-All contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
 
 ## Useful links
 
-* Discuss code changes of the Ballerina project
-  in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
+* For more information go to the [ASB package](https://central.ballerina.io/ballerinax/asb/latest).
+* For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
```

Comment:
Not sure this is valid for ASB connector. @NipunaRanasinghe any thoughts ?

Reply:
Shall we remove it then @NipunaRanasinghe ?  Since we don't have any BBs for ASB.